docs/specs/config.yml

```diff
@@ -454,23 +454,25 @@ outputs:
   b/a.json:
 ---
 # Peek .openpublishing.publish.config.json for docsets_to_publish and open_to_public_contributors
-inputs:
-  .openpublishing.publish.config.json: |
-    {
-      "docsets_to_publish": [
-        { "build_source_folder": ".", "open_to_public_contributors": true },
-        { "build_source_folder": "b", "open_to_public_contributors": true }
-      ]
-    }
-  docfx.yml: |
-    files: 'a.md'
-  a.md:
-  b/docfx.yml: |
-    globalMetadata:
-      open_to_public_contributors: false
-  b/b.md:
-  c/docfx.yml:
-  c/c.md:
+repos:
+  https://github.com/ops/config-1:
+  - files:
+      .openpublishing.publish.config.json: |
+        {
+          "docsets_to_publish": [
+            { "build_source_folder": ".", "open_to_public_contributors": true },
+            { "build_source_folder": "b", "open_to_public_contributors": true }
+          ]
+        }
+      docfx.yml: |
+        files: 'a.md'
+      a.md:
+      b/docfx.yml: |
```

Comment:
put another `.openpublishing.publish.config.json` under folder b? make sure it is not in use